package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "opentelemetry-azure-monitor-js",
+  "version": "0.0.0",
+  "description": "Application Insights exporter for the Open Telemetry JavaScript (Node.js) SDK",
```

Comment:
The Python repo currently is marketed as "exporter + SDK", meaning the solution does not only include logic for the exporter. Is this something we want to adopt as well for JS?

Reply:
This package.json would apply to the exporter only and would follow it wherever it goes. Once this repo morphs into a 'monorepo', a new root package.json will be created